### ISSUE TITLE
[high] Keyboard Navigation: About: 'Terms of service' and 'Privacy policy' links are not focusable

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5656]
## Action Performed:
Prerequisite: the user is logged in.

1. Using Chrome, open https://staging.new.expensify.com/
2. Navigate to the Settings > About.
3. Press Tab key and try to focus the links 'Terms of service' and 'Privacy policy'.

## Expected Result:
The 'Terms of service' and 'Privacy policy' links should be focusable by pressing Tab key.

## Actual Result:
The 'Terms of service', and 'Privacy policy' links are not focusable with Tab key.

HTML:
`<span class="css-901oao css-16my406 r-1loqt21" style="color: rgb(1, 133, 255); flex-shrink: 0; font-family: GTAmericaExp-Regular; font-size: 11px; height: 24px; line-height: 20px; text-decoration-color: rgb(1, 133, 255);">privacy policy</span>
<span class="css-901oao css-16my406 r-1loqt21" style="color: rgb(1, 133, 255); flex-shrink: 0; font-family: GTAmericaExp-Regular; font-size: 11px; height: 24px; line-height: 20px; text-decoration-color: rgb(1, 133, 255);">privacy policy</span>`

## Workaround:
No workaround

## Area issue was found in:
About overlay

## Failed WCAG checkpoints
2.1.1

## User impact:
The keyboard users won't be able to focus the elements.

## Suggested resolution:
Implement the elements as native links using `<a>` tag together with the non-empty href attribute, so that they can be focused with Tab key.
https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element
https://www.a11yproject.com/posts/2019-02-15-creating-valid-and-accessible-links/

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/136061200-16d35501-9c15-4bc4-a75a-f3d5d5d91e80.mp4

![Bug5185732_Keyboard_-_links_-_about](https://user-images.githubusercontent.com/88733897/136061224-35669478-ca32-4b96-8473-0b0454fe841f.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
